src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/List0FW.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+
+public class List0FW extends ListFW
+{
+    private static final DirectBuffer FIELDS_EMPTY_VALUE = new UnsafeBuffer(0L, 0);
+
+    private static final int LENGTH_SIZE = 0;
+
+    private static final int FIELD_COUNT_SIZE = 0;
+
+    private static final int LENGTH_OFFSET = 0;
+
+    private static final int FIELD_COUNT_OFFSET = LENGTH_OFFSET + LENGTH_SIZE;
+
+    @Override
+    public int limit()
+    {
+        return offset() + LENGTH_SIZE + length();
+    }
+
+    @Override
+    public int length()
+    {
+        return 0;
```

Comment:
Just for readability, suggest defining constant `LENGTH_VALUE = 0` and changing this to `return LENGTH_VALUE;`.